src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.WriteRaw.cs

```diff
@@ -450,22 +461,27 @@ private enum OverloadParamType
         {
             ROSChar,
             String,
-            ByteArray
+            ByteArray,
+            ROSeqByte
         }
 
-        private static void WriteRawValueWithSetting(Utf8JsonWriter writer, string payload, OverloadParamType param)
+        private static void WriteRawValueWithSetting(Utf8JsonWriter writer, string payload, OverloadParamType param, bool skipInputValidation = false)
         {
             switch (param)
             {
                 case OverloadParamType.ROSChar:
-                    writer.WriteRawValue(payload.AsSpan());
+                    writer.WriteRawValue(payload.AsSpan(), skipInputValidation);
                     break;
                 case OverloadParamType.String:
-                    writer.WriteRawValue(payload);
+                    writer.WriteRawValue(payload, skipInputValidation);
                     break;
                 case OverloadParamType.ByteArray:
                     byte[] payloadAsBytes = Encoding.UTF8.GetBytes(payload);
-                    writer.WriteRawValue(payloadAsBytes);
+                    writer.WriteRawValue(payloadAsBytes, skipInputValidation);
+                    break;
+                case OverloadParamType.ROSeqByte:
+                    ReadOnlySequence<byte> payloadAsSequence = new(Encoding.UTF8.GetBytes(payload));
+                    writer.WriteRawValue(payloadAsSequence, skipInputValidation);
```

Comment:
small suggestion: you might wanna add Debug.Fail in the default case of switch statement to future proof this code